src/pretix/control/templates/pretixcontrol/auth/components.html

```diff
@@ -0,0 +1,32 @@
+{% extends "pretixcontrol/auth/base.html" %}
+{% load bootstrap3 %}
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+<div class="container">
+    <div class="form-wrapper">
+        <div class="form-signin">
+            <h3>{% trans "COMPONENTS-PAGE.." %}</h3>
+            <h2>{% trans "THIS IS THE EVENTYAY-COMPONENTS PAGE" %}</h2>
+            <p>
+                {% blocktrans trimmed with a_attr="href='https://eventyay.com'" %}
+                    This is a self-hosted installation of <a {{ a_attr }}>eventyay, your open source event solution.</a>.
+                {% endblocktrans %}
+            </p>
+
+            {% if is_eventyay_talk_installed %}
+                <p>
+                    <a href="http://127.0.0.1:8000/eventyay-talk" class="btn btn-primary">
```

Comment:
Why is the hostname hardcoded as "127.0.0.1:8000"? It will not work in live system.

Reply:
had put this while testing,i had corrected it now.